addons/common/XEH_postInit.sqf

```diff
@@ -71,6 +72,20 @@
     };
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(blockRadio), {
+    params ["_object", "_set"];
+    TRACE_2("blockRadio EH",_object,_set);
+    if (_object isEqualTo ACE_Player && {_set > 0}) then {
+        call FUNC(endRadioTransmission);
+    };
+    if (isClass (configFile >> "CfgPatches" >> "task_force_radio")) then {
+        _object setVariable ["tf_unable_to_use_radio", _set > 0, true];
+    };
+    if (isClass (configFile >> "CfgPatches" >> "acre_main")) then {
+        _object setVariable ["acre_sys_core_isDisabled", _set > 0, true];
```

Comment:
This blocks direct speaking, not just radios.
It's no fun for people handcuffed

Reply:
`ace_mouthgag` when?

Reply:
Are you sure that actually works? I could swear a handcuffed person was speaking fine this weekend on our session.

Reply:
I can confirm, I used `player setVariable ["acre_sys_core_isDisabled", true, true];` yesterday to prevent player's from speaking during parts of a Halloween mission

Reply:
Yeah, yesterday we had self surrendering players which were muted, the character mimic was set to talking but they were unable to be heard.

Reply:
Ah I see, radio block for captive is completely new with this PR. We could add another variable to ACRE2 that only disables radio. Or API functions, even better.